library/alloc/src/collections/btree/node.rs

```diff
@@ -182,7 +182,7 @@ impl<K, V> Root<K, V> {
     /// make that new node the root node, and return it. This increases the height by 1
     /// and is the opposite of `pop_internal_level`.
     pub fn push_internal_level(&mut self) -> NodeRef<marker::Mut<'_>, K, V, marker::Internal> {
-        let mut new_node = Box::new(unsafe { InternalNode::new() });
+        let mut new_node = box unsafe { InternalNode::new() };
```

Comment:
I would prefer not to introduce use of the box keyword which is sort of unstable and unclear in terms of specification if Box::new is working well for us.